addon/components/hyper-table-v2/filtering-renderers/common/ordering.ts

```diff
@@ -0,0 +1,31 @@
+import { action } from '@ember/object';
+import Component from '@glimmer/component';
+
+import TableHandler from '@upfluence/hypertable/core/handler';
+import { Column, OrderDirection } from '@upfluence/hypertable/core/interfaces';
+
+interface HyperTableV2FilteringRenderersOrderingArgs {
+  handler: TableHandler;
+  column: Column;
+  customOrderingOptions: { [key: string]: OrderDirection };
```

Comment:
nit:
why not call it `orderingOptions` too? that way we can tell the difference by checking if it's `this.args.orderingOptions` or `this.orderingOptions` :)